content/en/logs/explorer/_index.md

```diff
@@ -4,182 +4,194 @@ kind: documentation
 
-{{< img src="logs/explorer/log_side_panel_infra.gif" alt="Hub to APM" style="width:60%;">}}
+The configuration of the log table is stored alongside other elements of your troubleshooting context in [Saved Views][1]
 
+#### List Aggregates of Logs
 
-### Configure your troubleshooting context
+The columns displayed in list of aggregates are columns **derived from the aggregation**.
 
-Interact with the attributes names and values in the lower JSON section to:
+Results are sorted according to:
 
-- Add or remove a column from the logs table.
-- Append the search request with specific values (include or exclude)
+- Number of matching events per aggregate for **pattern** aggregation (default to descending: more to less),
+- Lexocigraphic order of the transaction id for **transaction** aggregation (default to ascending: A to Z).
 
-{{< img src="logs/explorer/side_panel_context.gif" alt="Side Panel context"  style="width:60%;">}}
+### Timeseries
 
-- Build or edit a facet or measure from an attribute. See [Log Facets][18].
+Visualize the evolution of a single [measure][2] (or a [facet][2] unique count of values) over a selected time frame, and (optionally) split by an available [facet][2].
 
-{{< img src="logs/explorer/side_panel_facets.gif" alt="Side Panel Facets"  style="width:60%;">}}
+The following timeseries Log Analytics shows the evolution of the **top 5 URL Paths** according to the number of **unique Client IPs** over the last month.
 
-### Share a log
+{{< img src="logs/explorer/timeseries.png" alt="timeserie example"  style="width:90%;">}}
 
-Use the **Share** button to share the log opened in side panel to other contexts.
+Choose additional display options for timeseries: the **roll-up interval**, whether you **display** results as **bars** (recommended for counts and unique counts), **lines** (recommended for statistical aggregations) or **areas**, and the **colorset**.
 
-- **Copy to clipboard** or `Ctrl+C` / `Cmd+C` copies the log JSON to your clipboard.
-- **Share Event** shares the log (along with the underlying view) with teammates through email, Slack, and more. See all [Datadog notification integrations][19] available.
+### Toplists
 
-{{< img src="logs/explorer/upper_log_panel.png" alt="Upper Log Panel"  style="width:50%;">}}
+Visualize the top values from a [facet][2] according to the chosen [measure][2]:
 
-## Troubleshooting Context
+The following Top List Log Analytics shows:
+The evolution of the **top 5 URL Paths** according to the number of **unique Client IPs** over the last month.
 
-### Search Filter
+{{< img src="logs/explorer/toplists.png" alt="top list example"  style="width:90%;">}}
```

Comment:
This image shows logs grouped by customer name versus url paths. I believe the above text may be duplicated from the timeseries section, can you confirm @pcarioufr?